src/index.js

```diff
@@ -12,24 +12,27 @@ const ExtractTextPlugin = require('extract-text-webpack-plugin')
  * -------------------------------------------------------------------------- */
 
 module.exports = (neutrino, options = {}) => {
-  // We want to start from a clean slate. This removes any existing "style"
-  // rule that may exist (present if using the default neutrino-preset-web).
-  neutrino.config.module.rules.delete('style')
-
+  const { loaderOptions = {}, pluginOptions = {} } = options
   const styleRule = neutrino.config.module
     .rule('style')
-    .test(options.test || /\.css$/)
+    .test(loaderOptions.test || /\.css$/)
 
   const loaders = ExtractTextPlugin.extract({
-    fallback: options.fallback || 'style-loader',
-    use: options.use || 'css-loader'
+    fallback: loaderOptions.fallback || 'style-loader',
+    use: loaderOptions.use || 'css-loader'
   })
```

Comment:
I think we would get more flexibility here as well if we utilized Object.assign (currently your implementation does not support the `publicPath` option):

```js
const loaders = ExtractTextPlugin.extract(Object.assign({
  fallback: 'style-loader',
  use: 'css-loader'
}, loaderOptions))
```